### PR TITLE
Use what should hopefully be the de facto rippled role for ansible

### PR DESCRIPTION
**Pull request by tdfischer**
As part of review, this should also be reviewed:

https://github.com/ripple/rippled-ansible-role

+r

### CODE REVIEW

File: plays/install-rippled.yml [hunk 1]
```diff
@@ -5,16 +5,19 @@
...
-      apt_key: url=http://mirrors.ripple.com/mirrors.ripple.com.gpg.key
-    - name: Install rippled
-      apt: update_cache=true name=rippled
+  roles:
+    - role: rippled
+      vars:
+        rippled_ssl_cert: ../skynet-keys/server.crt
+        rippled_ssl_key: ../skynet-keys/server.key
+        rippled_data_root: /mnt/rippled/db/
+        rippled_log_file: /mnt/rippled/log/debug.log
```
Comment (wilsonianb) [inline plays/install-rippled.yml]: Why not move to `/var/lib/rippled` and `/var/log/rippled`?

**Comment by wilsonianb** [comment]
I commented on some commits in https://github.com/ripple/rippled-ansible-role

File: plays/install-rippled.yml (hunk 1 above)
Comment (tdfischer) [inline plays/install-rippled.yml]: Wasn't sure if we wanted to jump to that yet. Was planning on a future commit.

**Comment by wilsonianb** [comment]
rippled-ansible-role comments:
https://github.com/ripple/rippled-ansible-role/commit/30f8cce46385194fea965f7cdefea48c368bc5c2
https://github.com/ripple/rippled-ansible-role/commit/50f3489a97afdefdacf96f8f6a35b474b8abdaac

**Comment by wilsonianb** [comment]
+r

**Comment by tdfischer** [comment]
LGTM :+1:

**Pull request merged**